examples/80_extending/example_extending_regression.py

```diff
@@ -48,8 +49,7 @@ def fit(self, X, y):
                                                           kernel=self.kernel,
                                                           gamma=self.gamma,
                                                           degree=self.degree,
-                                                          coef0=self.coef0,
-                                                          )
+                                                          coef0=self.coef0)
```

Comment:
Shouldn't there be a linebreak after the bracket after KernelRidge?

Reply:
I'm not sure what you mean by this
* Do you mean just a linebreak between the two expressions `self.estimator = ...` and `self.estimator.fit(...)`?
```
self.estimator = sklearn.kernel_ridge.KernelRidge(alpha=self.alpha,
                                                  kernel=self.kernel,
                                                  gamma=self.gamma,
                                                  degree=self.degree,
                                                  coef0=self.coef0)
                                                  
self.estimator.fit(X, y)
return self
```
* You prefer the linespace by having the hanging bracket as it was?
```
self.estimator = sklearn.kernel_ridge.KernelRidge(alpha=self.alpha,
                                                  kernel=self.kernel,
                                                  gamma=self.gamma,
                                                  degree=self.degree,
                                                  coef0=self.coef0,
                                                  )
self.estimator.fit(X, y)
return self
```

Reply:
I mean:
```
self.estimator = sklearn.kernel_ridge.KernelRidge(
    alpha=self.alpha,
    kernel=self.kernel,
    gamma=self.gamma,
    degree=self.degree,
    coef0=self.coef0,
)
                                                  
self.estimator.fit(X, y)
return self
```